addons/website_event_sale/controllers/main.py

```diff
@@ -55,14 +55,19 @@ def registration_confirm(self, event, **post):
 
         return res
 
-    def _add_event(self, event_name="New Event", context=None, **kwargs):
+    def _prepare_event_values(self, name, event_start, event_end, address_values=None):
+        values = super(WebsiteEventSaleController, self)._prepare_event_values(name, event_start, event_end, address_values)
+        default_event_ticket_ids = request.env['event.event.ticket']
```

Comment:
Just to be modular, we could set a default value to tickets if not already present in values.